db/seeds.rb

```diff
@@ -1,7 +1,58 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+
+user = User.where(email: 'jorge@testing.com').first_or_create(password: 'jorgeeats', password_confirmation: 'jorgeeats')
+user2 = User.create( email: 'george@testing.com',password: 'georgeeats', password_confirmation: 'georgeeats')
```

Comment:
I would have both users follow the syntax of the first one. For clarity it might be a good idea to call your variables `user1` and `user2`